### PR TITLE
Discord Server & Community Growth report

### PR DESCRIPTION
**Pull request by bsdlcook**
Signed-off-by: Lewis Cook <lcook@FreeBSD.org>

### CODE REVIEW

**Comment by koobs** [comment]
LGTM @bsdlcook 

Reviewed by: koobs
Signed-off-by: koobs

File: 2021q1/discord.adoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
+== Discord Server & Community Growth ==
+
+Link: link:https://wiki.freebsd.org/Discord[Discord Wiki Page]
+Link: link:https://discord.gg/RHprKbvWJN[Discord Invitation Link]
+
+Contact: Lewis Cook <lcook@FreeBSD.org>
+Contact: Kubilay Kocak <koobs@FreeBSD.org>
```
Comment (koobs) [inline 2021q1/discord.adoc]: @bsdlcook Could you bump Vincent up the order here

File: 2021q1/discord.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Contact: Vincent Milum Jr <freebsd@darkain.com>
```
Comment (koobs) [inline 2021q1/discord.adoc]: @bsdlcook Could you bump Vincent up the order here

File: 2021q1/discord.adoc [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+The FreeBSD project and community continues to grow, and in the last 2
+quarters, we established an official FreeBSD server on the popular Discord
+platform, as a complementary means for out-reach, support, collaboration
+and social connection for and between members of the FreeBSD community,
+new and old.
+
+Discord boasts broad accessibility and unique features that the
+the community can enjoy without the steeper learning curves associated
+with other support and social mediums, that can often deter or overwhelm
+newcomers. It also provides an opportunity to broaden FreeBSD reach
```
Comment (koobs) [inline 2021q1/discord.adoc:19]: `FreeBSD's` reach

**Comment by bsdlcook** [comment]
Done! thanks @koobs

**Pull request closed without merging**